pkgs/top-level/all-packages.nix

```diff
@@ -27778,12 +27778,18 @@ with pkgs;
 
   qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix {};
 
+  qgis-ltr-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped-ltr.nix {
+    withGrass = true;
+  };
+
   qgis-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped.nix {
-    withGrass = false;
+    withGrass = true;
   };
 
   qgis = callPackage ../applications/gis/qgis { };
 
+  qgis-ltr = callPackage ../applications/gis/qgis/ltr.nix { };
```

Comment:
Sorry, but the ordering annoys me here. Could you group `qgis-ltr` and `qgis-ltr-unwrapped` together?